public/app/features/search/hooks/useManageDashboards.ts

```diff
@@ -6,6 +6,8 @@ import { DELETE_ITEMS, MOVE_ITEMS, TOGGLE_ALL_CHECKED, TOGGLE_CHECKED } from '..
 import { manageDashboardsReducer, manageDashboardsState, ManageDashboardsState } from '../reducers/manageDashboards';
 import { useSearch } from './useSearch';
 
+export const GENERAL_FOLDER_ID = 0;
```

Comment:
There's also `constants` file in the search folder where this could be stored.

Reply:
Brilliant suggestion, will move it there right away!